docs/credits.md

```diff
@@ -222,57 +261,136 @@ Octopus Deploy is made possible thanks to the following great third-party produc
 | @swc/counter                                      | 강동윤                                                                       | [https://github.com/swc-project/pkgs](https://github.com/swc-project/pkgs)                                                                                                                                                  | [Apache-2.0](https://spdx.org/licenses/Apache-2.0.html)                                                             |
-| @swc/jest                                         | 강동윤                                                                       | [https://github.com/swc-project/jest](https://github.com/swc-project/jest)                                                                                                                                                  | [MIT](https://spdx.org/licenses/MIT.html)                                                                           |
-| @swc/plugin-emotion                               | 강동윤                                                                       | [+https://github.com/swc-project/plugins](https://github.com/swc-project/plugins)                                                                                                                                          | [Apache-2.0](https://spdx.org/licenses/Apache-2.0.html)                                                             |
-| @swc/register                                     | 강동윤                                                                       | [https://github.com/swc-project/register](https://github.com/swc-project/register)                                                                                                                                          | [Apache-2.0](https://spdx.org/licenses/Apache-2.0.html)                                                            [MIT](https://spdx.org/licenses/MIT.html)                                                                           |
+| @swc/jest                                         | 강동윤                                                                       | [https://github.com/swc-project/pkgs](https://github.com/swc-project/pkgs)                                                                                                                                                  | [MIT](https://spdx.org/licenses/MIT.html)                                                                           |
+| @swc/plugin-emotion                               | 강동윤                                                                       | [+https://github.com/swc-project/plugins](+https://github.com/swc-project/plugins)                                                                                                                                          | [Apache-2.0](https://spdx.org/licenses/Apache-2.0.html)                                                             |
```

Comment:
@borland is there any way of preventing this nasty `+` signs being included in links? It makes the link behave oddly. I don't know what is different about some of these dependencies that causes it, but if we can strip them out the links would work.

Reply:
https://app.shortcut.com/octopusdeploy/story/120073/credits-file-generator-needs-to-fix-git-https-urls-for-npm-packages